[BUG] Tox tests failing

**Describe the bug**  
The `tox` test in the github actions constantly fails due to an error in the `omi` tool (version 0.2.1): 

```
  .tox/py38-nocov-linux/lib/python3.8/site-packages/egon/data/metadata/__init__.py:2: in <module>
      from omi.dialects import get_dialect
  .tox/py38-nocov-linux/lib/python3.8/site-packages/omi/dialects/__init__.py:1: in <module>
      import omi.dialects.oep.dialect
  .tox/py38-nocov-linux/lib/python3.8/site-packages/omi/dialects/oep/__init__.py:1: in <module>
      from omi.dialects.oep.dialect import OEP_V_1_3_Dialect
  .tox/py38-nocov-linux/lib/python3.8/site-packages/omi/dialects/oep/dialect.py:5: in <module>
      from omi.dialects.oep.parser import JSONParser_1_3
  .tox/py38-nocov-linux/lib/python3.8/site-packages/omi/dialects/oep/parser.py:13: in <module>
      from metadata.v130.schema import OEMETADATA_V130_SCHEMA
  E   ModuleNotFoundError: No module named 'metadata.v130'
```

This is why the GH actions never check for `black` and `isort`

@khelfen or @nesnoj or @MoritzSchloesser: Do you have an idea if we need a specific version or sth.? I can also look for it, I was just wondering if you already know more. 

---

**To Reproduce**  
Steps to reproduce the behavior:
1. Push anything
2. See that GH actions are failing

--- a/setup.py
+++ b/setup.py
@@ -96,6 +96,7 @@ setup(
         "numpy<1.23",  # incompatibilities with shapely 1.7.
         # See: https://stackoverflow.com/a/73354885/12460232
         "oedialect==0.0.8",
+        "oemetadata<=2.0.1",
         "omi",
         "openpyxl",
         "pandas>1.2.0,<1.4",  # pandas>=1.4 needs SQLAlchemy>=1.4

--- a/setup.py
+++ b/setup.py
@@ -96,7 +96,7 @@ setup(
         "numpy<1.23",  # incompatibilities with shapely 1.7.
         # See: https://stackoverflow.com/a/73354885/12460232
         "oedialect==0.0.8",
-        "oemetadata<=2.0.1",
+        "oemetadata<=2.0.1", # incompatibilities with metadata 1.3
         "omi",
         "openpyxl",
         "pandas>1.2.0,<1.4",  # pandas>=1.4 needs SQLAlchemy>=1.4

